### PR TITLE
[misc] Lint and template updates

### PR DESCRIPTION
**Pull request by brendacs**
### Purpose of pull request

**Related Issue**: N/A

Add more eslint rules, fix lint script, update .github templates

### Pull request checklist

- [x] Branch and PR are named correctly
- [x] Updated relevant documentation
- [ ] Tested with a tester bot
- [ ] Wrote unit tests for changes

### Testing instructions

Running `yarn lint` should work

### CODE REVIEW

**Comment by brendacs** [comment]
> forgot to mention, 2 indent?

have objection? can switch to 4, don't really mind

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
+
 If applicable, add screenshots to help explain your problem.
 
 ### Environment (please complete the following information):
- - _OS_: [e.g. iOS, Windows, MacOS]
- - _Browser_: [e.g. chrome, safari, iOS/android app]
- - _Version_: [e.g. 22]
- - _Device_: [e.g. iPhone6]
+
+- _OS or Device_: [e.g. iOS, Windows, MacOS, iPhone 6]
```
Comment (sirNikolai) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: Might be a bit redundant to have `iOS` if iPhone 6 is there. I guess for phones it's less important of OS as much as app version is.

```suggestion
- _OS or Device_: [e.g. Windows, MacOS, iPhone 6]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
+- _Browser_: [e.g. Chrome, Safari, iOS/Android]
+- _Version_: [e.g. 22]
```
Comment (sirNikolai) [inline .github/ISSUE_TEMPLATE/bug_report.md:34]:
```suggestion
- _App Version_: [e.g. 22]
```

**Review by sirNikolai — APPROVED** [review]
lgtm, just some minor pedantic things.

**Review by sirNikolai — APPROVED** [review]

**Pull request merged**